common/models/access-token.js

```diff
@@ -101,7 +101,12 @@ module.exports = function(AccessToken) {
         if (err) {
           cb(err);
         } else if (token) {
-          token.validate(function(err, isValid) {
+          var possibleDate = new Date(token.created);
+          var isAValidDate = !isNaN(possibleDate.getTime());
+          if (isAValidDate) {
+            token.created = new Date(token.created);
+          }
```

Comment:
This looks suspicious. What part of the system is producing `token.created` that is not already a `Date` instance? I'd rather fix the problem at the source.

/cc @ritch @kraman